x-pack/heartbeat/monitors/browser/synthexec/synthexec_test.go

```diff
@@ -215,10 +214,10 @@ func eventsWithType(typ string, synthEvents []*SynthEvent) (matched []*SynthEven
 
 func TestProjectCommandFactory(t *testing.T) {
 	_, filename, _, _ := runtime.Caller(0)
-	origPath := path.Join(filepath.Dir(filename), "../source/fixtures/todos")
+	origPath := filepath.Join(filepath.Dir(filename), "..", "source", "fixtures", "todos")
```

Comment:
more incorrectness that was due to be fixed, though it's a bit pedantic I shoehorned it in